### DISCUSSION TITLE
Efficient implementation of merge duplicates/de-duplicates.

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [retorquere/zotero-better-bibtex#2505] [General]
I noticed that merging duplicate items in zotero is very time-consuming, even with [this plugin](https://github.com/frangoud/ZoteroDuplicatesMerger). So, I wonder whether this tool has the feature of efficient implementation of merge duplicates/de-duplicates?

Regards,
Zhao

**Comment by retorquere** [comment]
I don't see how BBT could be involved in de-
duplication.
